plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -52,8 +52,8 @@ func getAllTolerationPreferNoSchedule(tolerations []v1.Toleration) (tolerationLi
 	return
 }
 
-func getTolerationListFromPod(pod *v1.Pod) ([]v1.Toleration, error) {
```

Comment:
I agree with Avesh. The fact that this wrapper drops "PreferNoSchedule" from the name of the wrapped function gives the impression to the reader that the function returns all of the pod tolerations.